.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["env"],
+  "plugins": ["angularjs-annotate"]
```

Comment:
this file added to bring in the angular-annotate plugin, which replaces the ng-annotate plugin, which replaces the ng-min plugin